runtime/src/bank_forks.rs

```diff
@@ -100,16 +106,14 @@ impl BankForks {
 
         let parents = root_bank.parents();
         for parent in parents {
-            if banks
-                .insert(
-                    parent.slot(),
-                    BankWithScheduler::new_without_scheduler(parent.clone()),
-                )
-                .is_some()
-            {
-                // All ancestors have already been inserted by another fork
```

Comment:
This is leftover from when `BankForks` could be initialized with a list of banks.